utils/RunONNXModel.py

```diff
@@ -550,6 +550,9 @@ def warning(msg):
 def data_without_top_bottom_quartile(data, percent):
     data = np.array(sorted(data))
     trim = int(percent * data.size / 100.0)
+    if trim == 0 or data.size - 2 * trim < 1:
+        # Want at least one element, return as is.
+        return data
```

Comment:
Thanks! I sometimes got the error here with a small number of iterations, say 3.

Reply:
So did I :-)